source/crud/crud.rst

```diff
@@ -11,8 +11,8 @@ Driver CRUD API
 :Advisors: Jeremy Mikola, Jeff Yemin
 :Status: Approved
 :Type: Standards
-:Minimum Server Version: 2.6
-:Last Modified: October 28, 2019
+:Minimum Server Version: 2.7
+:Last Modified: November 20, 2019
```

Comment:
Nit: update this to whatever date it happens to be when the PR is merged.

Reply:
Done